nat-gatway.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_eip" "ip" {
```

Comment:
Can we name it something that identifies this as specific to the private subnet? 

e.g 
```
resource "aws_eip" "privates_subnet_nat_gateway_public_ip" 
```